### PR TITLE
add documentation for the copy cell content action

### LINKED ISSUE #1104: [FEATURE] add copy cell content action in attributetable context menu
Original commit: https://github.com/qgis/QGIS/commit/3da3cf18b536c0949f771e41eb5f26c2ce6dca72 by nyalldawson

Unfortunately this naughty coder did not write a description... :-(

### PR DESCRIPTION
**Pull request by pblottiere**
This PR add documentation for the copy cell content action.

Note that this feature is for 2.16.

Fix #1104

### CODE REVIEW

**Comment by yjacolin** [comment]
This PR will probably have a conflict. Does your local master is up to date with the one in QGIS repository? Attribute table documentation has been move to another file.

See https://github.com/qgis/QGIS-Documentation/pull/1127

**Comment by yjacolin** [comment]
Ok the PR was not already merged. Sorry @pblottiere

File: source/docs/user_manual/working_with_vector/editing_geometry_attributes.rst [hunk 1]
```diff
@@ -1034,6 +1034,24 @@ Zoom to feature
...
+.................
+
+To copy the content of a cell in the clipboard, right-click on the feature within
+the attribute table and select :guilabel:`Copy cell content`.
+
+.. _figure_copy_cell_content:
+
+.. only:: html
+
+  **Figure Copy cell content:**
```
Comment (DelazJ) [inline source/docs/user_manual/working_with_vector/editing_geometry_attributes.rst]: What about "**Figure Attributes xxx:**" as it's usually done for figures? (same logic for the replacement above)

**Comment by DelazJ** [comment]
@pblottiere thank you for always documenting your features

**Comment by yjacolin** [comment]
@pblottiere could you rebase your PR from master? It is time to merge your contribution :)

**Pull request merged**